docs/components.rst

```diff
@@ -86,6 +86,57 @@ socket for the duplicator.
 
 .. _msgpack: https://msgpack.org/index.html
 
+.. _receiver-component:
+
+Receiver
+--------
+
+The mite receiver component is a generic mechanism that can perform tasks
+not limited to fulfilling a single role (eg. stats, collector). This allows
+changing a mite pipeline more easily, without introducing new processes and
+network components. A ``mite.cli.receiver`` instance can have multiple custom
+``processors`` connected to it as either listeners or raw listeners and can
+dispatch incoming messages to them.
+
+.. code-block:: sh
+
+   POD_NAME=1 mite receiver tcp://127.0.0.1:14303 \
```

Comment:
Why is POD_NAME needed?

Reply:
it's tied to the CollectorProcessor example class beneath which does `self.collector = collector.Collector(collector_id=os.environ["POD_NAME"])` and I wanted the example to be complete so I put it in... should I take it out ?

Reply:
Yeah, I think it's confusing.  Can the collector id just be a random uuid?  (Or a random unsigned integer, if it is constrained to being an integer?)  @johnarnfield would that work?

Reply:
It would 👍

Reply:
I changed the example code to use a uuid